pymongo/asynchronous/command_cursor.py

```diff
@@ -381,21 +385,28 @@ async def __aenter__(self) -> AsyncCommandCursor[_DocumentType]:
     async def __aexit__(self, exc_type: Any, exc_val: Any, exc_tb: Any) -> None:
         await self.close()
 
-    async def to_list(self) -> list[_DocumentType]:
+    async def to_list(self, length: int = -1) -> list[_DocumentType]:
```

Comment:
Why use -1? I was thinking we'd use `None` to mean no limit to match Motor. Is it for nicer typing (since there's no need for Optional)?

Reply:
I'll switch to Optional[int].